sr_example/scripts/sr_example/hand_examples/sr_right_hand_follow_finger_trajectory.py

```diff
@@ -15,60 +15,74 @@
 prefix = hand_parameters.mapping.values()[0]
 hand_serial = hand_parameters.mapping.keys()[0]
 
-# Open fingers
-group_id = prefix + "_fingers"
-hand_commander = SrHandCommander(name=group_id)
-hand_commander.move_to_named_target("fingers_open")
+# Close all fingers except thumb
+hand_commander = SrHandCommander()
+hand_commander.move_to_named_target("fingers_pack_thumb_open")
 
+# Change to first finger group
 group_id = prefix + "_first_finger"
 hand_commander = SrHandCommander(name=group_id)
 
-rospy.sleep(2)
+# Open first finger
+hand_commander.move_to_named_target("first_finger_open")
+
+rospy.sleep(4)
+
 rospy.loginfo("\nMoving Index finger following a set of waypoints")
 
 waypoints = []
 
+# Move horizontally
+start_pose = geometry_msgs.msg.Pose()
+start_pose.position.x = 0.034
+start_pose.position.y = -0.002
+start_pose.position.z = 0.197
+start_pose.orientation.w = 1.00
```

Comment:
As discussed with @beatrizleon  the orientation is ignored by the IK for the fingers, as most orientations would be infeasible anyway. This will be documented.